lib/aika_web/templates/layout/app.html.eex

```diff
@@ -27,6 +27,15 @@
       <div class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></div>
     <% end %>
 
+    <div class="api-token">
```

Comment:
Could this move to the dashboard index page for now instead as I believe this layout is still used by pages such as the signup page, password reset etc

Also could this be a shared partial instead, feels like a nice neat component